### DISCUSSION TITLE
[Node-Red] How do I play sound alerts via DF Player

### DISCUSSION THREAD

**Discussion by Techposts** [pixelit-project/PixelIt#161] [Q&A] [answered]
Following is the connection. No Sound from the output. Connected and DF Player LED is ON.

![image](https://user-images.githubusercontent.com/28442414/159645160-d32e2d69-3f1f-4de0-9c67-0a026e6cf20d.png)
![image](https://user-images.githubusercontent.com/28442414/159645687-76b598b1-d944-48e8-ae79-c2c7910b8549.png)

**Comment by Techposts** [comment, marked as answer]
Also, why Node Red crashes every once in a while
![image](https://user-images.githubusercontent.com/28442414/159649462-f367efdf-8bb3-41d9-9653-1e36c417ef59.png)

**Reply by o0shojo0o** [reply]
Also a sound is treated like a screen, so it must be sent to "Alert Screen" or "Screen Data Update".

**Reply by Techposts** [reply]
Awesome. This worked. Connected to the alert screen and it worked. 
![image](https://user-images.githubusercontent.com/28442414/159653653-0558cfed-3021-4f7b-9189-ac93fadddc22.png)
